Sources/SwiftKueryORM/Model.swift

```diff
@@ -522,7 +522,7 @@ public extension Model {
 
     /// - Parameter using: Optional Database to use
```

Comment:
I appreciate the docs need an overhaul, but could we add at least a one-liner explaining what these functions are for? eg:
> Allows custom functions on a model to query the database directly.

Reply:
Done.